code/style.css

```diff
@@ -1,4 +1,275 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
 }
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  color: #000000;
+}
+
+/* ---- Logo & Navbar ---- */
+nav {
+  background: #ffffff;
+  padding: 0 15px;
+  display: grid;
+  grid-template-columns: 100%;
+  width: 100%;
+  height: 200px;
+}
+
+.menu {   
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu li a {
+  display: inline;
+  flex-direction: row;
+  padding: 30px 5px;
+  margin-left: 80px;
+  display: none;
+}
+
+.logo {
+  position: relative;
+  width: 300px;
+  flex-direction: row;
+  margin-left: 350px;
+  margin-top: 20px;
+}
+
+a {
+  color: #f0b616;
+  text-decoration: none;
+  font-weight: bold;
+  margin-left: 20px;
+}
+
+a:hover {
+  background: gray;
+  transition: .5s;
+  border-radius: 20px;
+}
```

Comment:
The links are beautiful! I really like the effect you have when you hover over them!